guacamole-ext/src/main/java/org/apache/guacamole/properties/GuacamoleProperty.java

```diff
@@ -49,5 +51,25 @@
      *                            provided value.
      */
     public Type parseValue(String value) throws GuacamoleException;
+    
+    /**
+     * Parses the given string value into a sorted Collection of values of the
+     * type associated with this GuacamoelProperty.
+     * 
+     * @param value
+     *     The string value to parse.
+     * 
+     * @return
+     *     A sorted Collection of the parsed values.
+     * 
+     * @throws GuacamoleException 
+     *     If an error occurs while parsing the provided value.
+     */
+    default public Collection<Type> parseValueCollection(String value)
```

Comment:
We should note the default behavior in the documentation such that implementations that inherit this implementation will have that behavior documented. It may otherwise be surprising to a developer invoking `parseValueCollection()` that the property they're using doesn't support being parsed as a collection.